stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -69,7 +147,176 @@ private static String getIdAttribute(View element) {
 
   @Override
   public View getViewForHighlighting(Object element) {
-    return (View)element;
+    return (View) element;
+  }
+
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+
+    for (ViewCSSProperty property : getProperties()) {
```

Comment:
Have you considered making `mViewProperties` an array instead of a `Set`? I don't think we actually need the set mechanics, and it would allow us to avoid an allocation here in the for-each block. And since `HashSet<T>` is implemented as a wrapper around a `HashMap<T, null>', we end up with an additional allocation for every item we put into it (`Map.Entry`).